mne_bids/tests/test_utils.py

```diff
@@ -34,6 +34,40 @@
     task=task)
 
 
+@pytest.fixture(scope='session')
+def return_bids_test_dir(tmpdir_factory):
+    """Return path to a written test BIDS dir."""
+    output_path = str(tmpdir_factory.mktemp('mnebids_utils_test_bids_ds'))
+    data_path = testing.data_path()
+    raw_fname = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc_raw.fif')
+
+    event_id = {'Auditory/Left': 1, 'Auditory/Right': 2, 'Visual/Left': 3,
+                'Visual/Right': 4, 'Smiley': 5, 'Button': 32}
+    events_fname = op.join(data_path, 'MEG', 'sample',
+                           'sample_audvis_trunc_raw-eve.fif')
+
+    raw = mne.io.read_raw_fif(raw_fname)
+    with pytest.warns(UserWarning, match='No line frequency'):
+        write_raw_bids(raw, bids_basename, output_path,
+                       events_data=events_fname, event_id=event_id,
+                       overwrite=False)
+
+    return output_path
+
+
+def test_get_list_of_entity(return_bids_test_dir):
+    """Test getting a list of entities."""
+    bids_root = return_bids_test_dir
+    with pytest.raises(ValueError, match='Key must be one of'):
+        get_list_of_entity(bids_root, key='bogus')
+
+    assert get_list_of_entity(bids_root, 'sub') == [subject_id]
+    assert get_list_of_entity(bids_root, 'ses') == [session_id]
+    assert get_list_of_entity(bids_root, 'run') == [run]
+    assert get_list_of_entity(bids_root, 'acq') == [acq]
```

Comment:
what happens if there is no `acq` key? or if there are more than one run? should we add a test for those cases?